backend/src/cron/email.ts

```diff
@@ -25,12 +27,22 @@ export class EmailCron implements Cron {
     async start(): Promise<void> {
         let enabled = await isEmailCronEnabled()
         if (enabled) {
-            this.job = cron.schedule(this.options.cronExpression, () => {
+            this.job = cron.schedule(this.options.cronExpression, async () => {
                 if (this.client === EmailClient.AWS) {
                     console.log('Sending email with AWS')
+                    // enable later 
+                    if (false) {
```

Comment:
A false check here is not required. But if you want to keep a check then use the  feature flag method by adding flags in the config files.